app/org/sagebionetworks/bridge/models/Email.java

```diff
@@ -1,31 +1,25 @@
 package org.sagebionetworks.bridge.models;
 
-import org.sagebionetworks.bridge.json.JsonUtils;
 import org.sagebionetworks.bridge.models.studies.StudyIdentifier;
 import org.sagebionetworks.bridge.models.studies.StudyIdentifierImpl;
 
-import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.annotation.JsonProperty;
 
 public class Email implements BridgeEntity {
 
-    private static final String EMAIL_FIELD = "email";
-    private static final String STUDY_FIELD = "study";
-    
     private final String email;
     private final StudyIdentifier studyIdentifier;
     
-    public Email(StudyIdentifier studyIdentifier, String email) {
-        this.studyIdentifier = studyIdentifier;
+    public Email(@JsonProperty("study") String study, @JsonProperty("email") String email) {
+        this.studyIdentifier = (study != null) ? new StudyIdentifierImpl(study) : null;
         this.email = email;
     }
-    
-    public static final Email fromJson(JsonNode node) {
-        String email = JsonUtils.asText(node, EMAIL_FIELD);
-        String study = JsonUtils.asText(node, STUDY_FIELD);
-        StudyIdentifier studyIdentifier = (study == null) ? null : new StudyIdentifierImpl(study);
-        return new Email(studyIdentifier, email);
-    }
 
+    public Email(StudyIdentifier studyId, String email) {
+        this.studyIdentifier = studyId;
```

Comment:
Email has an EmailValidator while EmailVerification does not. So I should add that. For these trivial classes it would be nicest to have a validator that used annotations and that was reusable for all these classes.